### ISSUE TITLE
"Are you sure you want to delete" messages are misleading

### ISSUE THREAD

**Issue by axfelix** [fsulib/islandora_managed_access#6]
Trying to remove managed access permissions from an object via the module's configuration menu implies that the object is being deleted outright, which I don't think is the case -- could stand a rewording here.

**Comment by bryjbrown** [comment]
Good catch. The names of the entities involved in this module are inherently confusing (I found that out when trying to write documentation that isn't technical gibberish), its just hard to name things well. I'll see if I can find another way to word this.
